src/dotnet/APIView/APIViewWeb/Languages/CppLanguageService.cs

```diff
@@ -121,7 +121,7 @@ public override async Task<CodeFile> GetCodeFileAsync(string originalName, Strea
             astStream.Position = 0;
 
             //Generate pacakge name from original file name
-            var packageNamespace = originalName.Substring(0,originalName.LastIndexOf(".")).Replace("_", "::");
+            var packageNamespace = originalName.Substring(0,originalName.LastIndexOf(".")).Replace("_", "::").ToLower();
```

Comment:
Because internally the namespace is internally case sensitve we should store the package name with the casing.  Rather than doing this I suggest using the dropdown selection in the UI.

As a stopgap you should add a regex to the package name that the file upload has format
[A-Z][A-Za-z]*_........

```suggestion
            var packageNamespace = originalName.Substring(0,originalName.LastIndexOf(".")).Replace("_", "::");
```